xbmc/peripherals/bus/virtual/PeripheralBusAddon.h

```diff
@@ -34,12 +34,18 @@
 
 namespace PERIPHERALS
 {
-  class CPeripheralBusAddon : public CPeripheralBus
+  class CPeripheralBusAddon : public CPeripheralBus, public ADDON::IBinaryAddonManagerCallback
```

Comment:
The CAddonManager events mechanism is not usable, it allow only non blocking calls. 

Further is wanted from @tamland to remove the old callback system with IAddonMgrCallback and the way over the addons (what is good) and go only over "CEventSource".